### ISSUE TITLE
Random letters turn grey when typing

### ISSUE THREAD

**Issue by UnprecedentedWolf** [wereturtle/ghostwriter#743]
**Platform:**
Windows 10

**ghostwriter installation:**
ghostwriter 2.1.1 portable

**Describe the bug**
When I try to type or paste in a text, some letters get a different color - grey on default classic theme and light blue on plainstraction. For example, in the following text:
```
mamma mia
arigatou
family
green green
hi hello
```
first "m", as well as "i" in "arigatou" have a different color. Typing more may result in additional letters changing color - sometimes not immediately, but rather only after moving the cursor to it.

**To Reproduce**
1. Open a new file in Ghostwriter
2. Copy nad paste the text quoted above
3. Notice that some letters have different color

**Expected behavior**
When no markdown tags are used, all symbols should have the same color.

**Screenshots**
![image](https://user-images.githubusercontent.com/79556012/158090964-9b510184-e069-44cd-b493-c197b4c8a679.png)

**Comment by abenson** [comment]
Duplicate of #624?  Should be fixed in 2.1.2.

**Comment by UnprecedentedWolf** [comment]
My bad, missed that when searching for existing issues.
